### DISCUSSION TITLE
Need more infomation about captive-portals.txt .

### DISCUSSION THREAD

**Discussion by lessload** [DNSCrypt/dnscrypt-proxy#1842] [Q&A] [answered]
Need more infomation about `captive-portals.txt`. 
How different between `cloaking-rules` and `captive-portals` ?
This rule was overwrite cloak rule ?
Why use multiple text format in one program ? it so confused. seem like `url x.x.x.x, x.x.x.x` not work in other rule.

**Comment by jedisct1** [comment, marked as answer]
I found this in the documentation: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Windows-NCSI
